packages/sdk-metrics/src/aggregator/exponential-histogram/mapping/ExponentMapping.ts

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as ieee754 from './ieee754';
+import * as util from '../util';
+import { Mapping, MappingError } from './types';
+
+/**
+ * ExponentMapping implements a exponential mapping functions for
+ * for scales <=0. For scales > 0 LogarithmMapping should be used.
+ */
+export class ExponentMapping implements Mapping {
+  static readonly MIN_SCALE = -10;
+  static readonly MAX_SCALE = 0;
+  private static readonly _PREBUILT_MAPPINGS = [
+    new ExponentMapping(10),
+    new ExponentMapping(9),
+    new ExponentMapping(8),
+    new ExponentMapping(7),
+    new ExponentMapping(6),
+    new ExponentMapping(5),
+    new ExponentMapping(4),
+    new ExponentMapping(3),
+    new ExponentMapping(2),
+    new ExponentMapping(1),
+    new ExponentMapping(0),
+  ];
```

Comment:
our browser friends would probably appreciate the minification benefits of doing this in a descending loop.

```suggestion
  private static readonly _PREBUILT_MAPPINGS = Array(11)
    .fill(10)
    .map((v, i) => new ExponentMapping(v - i))
```